machine/quanta/quanta_ly4r/kernel/fix-i2c-ismt-write-error.patch

```diff
@@ -0,0 +1,33 @@
+fix i2c-ismt write error
+
+diff --git a/drivers/i2c/busses/i2c-ismt.c b/drivers/i2c/busses/i2c-ismt.c
+index 6e932d1..7dd87b8 100644
+--- a/drivers/i2c/busses/i2c-ismt.c
++++ b/drivers/i2c/busses/i2c-ismt.c
+@@ -675,6 +675,23 @@ static irqreturn_t ismt_do_interrupt(int vec, void *data)
+  */
+ static irqreturn_t ismt_do_msi_interrupt(int vec, void *data)
```

Comment:
We use i2c-ismt as /dev/i2c-1, which we put board info eeprom here.
We found the i2cset command did not complete normally, Therefore we added the while loop to wait the in progress flag complete, then clear the interrupt flags.